src/control/server/faults.go

```diff
@@ -50,6 +50,11 @@ var (
 		"cannot create a pool without a pool label",
 		"retry the operation with a label set",
 	)
+	FaultPoolHasContainers = serverFault(
+		code.ServerPoolHasContainers,
+		"cannot create a pool with existing containers",
+		"retry the operation with the -r|--recursive flag set",
```

Comment:
typo: s/create/destroy/

Also, the resolution string shouldn't use CLI-specific terms, as there are other potential users of the Control API than dmg. I suggest something more like, "set the recursive flag to remove containers along with the pool".

Reply:
thanks, done